### ISSUE TITLE
Commerce Variant  Field only searches by variant title, not product title

### ISSUE THREAD

**Issue by bossanova808** [craftcms/commerce#934]
### Description

This is a UI issue, but when I am searching for a variant like

'Clearbags - small'

(which is the product title - variant title)

...I'd expect to be able to start the search with `clearbags`, but that gives no results.  So I have to search on `small` which then brings up all sorts of unrelated small products, and is counter-intuitive for authors I feel.

(
...I may be I am missing some sort of facility for this to do with (auto) variant titling?  Ideally it would use the Product Title - Variant Title, and or the Order Description Format (e.g. for these I have 

```
{% if object.product.type.hasVariants %}{{ object.product.title }} - {{ object.title}}{% else %}{{ object.title}}{% endif %}
```
)

**Comment by lukeholder** [comment]
I will add the description to the search index, that should solve it. Adding to the list.

**Comment by nfourtythree** [comment]
This has been updated and will be included in the `2.2` release
